lib/Cache/RuntimeCache.php

```diff
@@ -24,6 +24,8 @@ class RuntimeCache extends \ArrayObject
 
     protected static ?RuntimeCache $instance = null;
 
+    public static bool $disabled = false;
```

Comment:
```suggestion
    private static bool $disabled = false;
```
Why does this have to be public when you have the functions to access it?